code/datums/mutations/_mutations.dm

```diff
@@ -1,6 +1,17 @@
 /datum/mutation
 	var/name
 
+/*
+
+NEGATIVE STABILITY LEGACY
+per 5/24/2024 rework, negative mutations give stability! via, roughly, these rules:
+
+-5~: For barely negative mutations that are often injected just for 'fun'. Eg: Tourettes, Swedish.
```

Comment:
I would keep voice affecting mutations at 0 instab still. Don't really wanna encourage people roleplaying less to get free powers.